### PR TITLE
Add first-pass LD context and definitions docs for SVIP use-cases

### PR DESCRIPTION
**Pull request by hannemennah**
Moving the rough drafts of schemas and definitions from our private self-hosted repository to a publicly-visible repo.

### CODE REVIEW

**Comment by glcohen** [comment]
Hmm, I'm a little confused. I don't see any json schemas, just LD. Is that intentional?

**Comment by OR13** [comment]
Agreed there are no json schema definitions here.

Here is the schema definition for our certified mill test report: https://github.com/w3c-ccg/vc-json-schemas/blob/master/docs/schema/steel-mill-cert-test-report.json 

You can see it referenced in the example credential here: https://github.com/w3c-ccg/vc-json-schemas/blob/master/docs/credentials/steel-mill-cert-test-report.example.vc.jsonld#L14

Since you appear to have a lot of credentials that will need schema definitions, and no test code that produces those credentials in this PR, I would suggest you add a simple schema for one of the smaller credential formats.

You can then sign that schema using the following code: https://github.com/w3c-ccg/vc-json-schemas/blob/master/hypothetical-use-cases/trade-certificates/examples/steel-mill-cert-test-report/create-vc-json-schema.spec.js

And then produce a signed credential that uses that schema using this code: https://github.com/w3c-ccg/vc-json-schemas/blob/master/hypothetical-use-cases/trade-certificates/examples/steel-mill-cert-test-report/create-vc.spec.js

Then place your signed schema and example credential in the docs directory as exemplars.

**Comment by OR13** [comment]
If you intend to use the latest did context with JWK formatted keys, you are blocked by: https://github.com/w3c/did-core/pull/156

File: docs/context/hypothetical-examples/handoff-v1.jsonld [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "@context":
+    {
+      "@version": 1.0,
+      "handoff": "https://project.factom.com/contexts/handoff-v1#",
+      "xsd": "http://www.w3.org/2001/XMLSchema#",
+      "schema": "http://schema.org/",
+      "@type": "schema:TransferAction",
+      "recipient": "schema:recipient",
```
Comment (mocha384) [inline docs/context/hypothetical-examples/handoff-v1.jsonld]: @hannemennah 
This handoff document looks good and Mavennet will also be using it for interoperability purposes. 
In order to adapt it for the pipeline use case, we would like to add more fields like

1. delivery volume
2. Route (which pipeline - the number or name)
3. Nomination month (a time when it happened)

Let us know what you think about it.

**Comment by OR13** [comment]
@hannemennah I merged mavennets PR, as they included a json schema. This PR is blocked because your example does not contain a JSON Schema.

**Comment by OR13** [comment]
@hannemennah any thought on this? I'd like to merge it when possible

**Comment by glcohen** [comment]
Please reopen if still interested.

**Pull request closed without merging**